src/org/robockets/steamworks/Robot.java

```diff
@@ -139,13 +151,14 @@ public void teleopInit() {
 			autonomousCommand.cancel();
 
 		drive.start();
-
-		SmartDashboard.putNumber("GyroP", drivetrain.gyroPID.getP());
-		SmartDashboard.putNumber("GyroI", drivetrain.gyroPID.getI());
-		SmartDashboard.putNumber("GyroD", drivetrain.gyroPID.getD());
-		SmartDashboard.putNumber("GyroSetpoint", drivetrain.gyroPID.getSetpoint());
-
+		
+		SmartDashboard.putNumber("drivetrain setpoint", drivetrain.encoderPID.getSetpoint());
```

Comment:
Will be fixed when driving-pid-iterator and driving-pid is merged into drivetrain.